### ISSUE TITLE
Example code ledc_basic, there is a strange problem. Maybe it's a bug. (IDFGH-7059)

### ISSUE THREAD

**Issue by SuhaylZhao** [espressif/esp-idf#8676]
Hello everyone，

I need help，I ran this example, and it's very normal.
https://github.com/espressif/esp-idf/tree/master/examples/peripherals/ledc/ledc_basic
![image](https://user-images.githubusercontent.com/3676296/160524902-bec8f07e-4e56-455d-aee1-78bcd3843dfc.png)

But when I changed gpio5 to gpio33, a strange problem occurred. The low level of PWM output is not 0V, but 1.65v, which is about 50% of VCC. Ask for help. Do I need any settings?

![image](https://user-images.githubusercontent.com/3676296/160524714-5fe01f64-5da6-4a6b-8ca6-2bda42b9530a.png)

**Comment by songruo** [comment]
Hi @SuhaylZhao,

Please provide more details about the module/chip you are using, and the Kit version (for WroverKit/PicoKit/DevKitC).

**Comment by SuhaylZhao** [comment]
Hi @songruo ，

The Module is Wrover-E.

Chip is ESP32-D0WD-V3 (revision 3)
Features: WiFi, BT, Dual Core, 240MHz, VRef calibration in efuse, Coding Scheme None
Crystal is 40MHz
MAC: 44:17:93:e3:77:70

Thanks！ 

In addition, I found that this may be a hardware problem. I use another esp32 board and burn the same code. Gpio33 pin can also output normal PWM waveform. 

But I still feel confused. Why is there such a problem? In one of my products, I connected gpio33 to a buzzer, so I urgently want to know the answer to the problem.

**Comment by songruo** [comment]
@SuhaylZhao Yes, I understand this is the module you are using. But what about the development kit (the one has the module attached on)? If there is a 32kHz crystal, it will be on the development kit board (like [this one](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/esp32/get-started-wrover-kit.html?highlight=esp32%20wrover%20kit#functional-description)). Or are you developing your own board for the module?

**Comment by SuhaylZhao** [comment]
Hi @songruo ,

I use freenove's esp32-wrover-dev, and there is no crystal oscillator outside.

![image](https://user-images.githubusercontent.com/3676296/160576391-d1ec4339-8b16-4218-92d8-a6de5990a61e.png)
![IMG_2440](https://user-images.githubusercontent.com/3676296/160576969-84935903-5ddb-4ec4-89f4-0cfdb9ccdf77.JPG)

**Comment by songruo** [comment]
@SuhaylZhao I tried but failed to find freenove esp32_wrover_dev board circuit diagram. But as you said, this should be a hardware problem with the development board. If you want to figure out the reason, you probably need to ask freenove for the schematic.

**Comment by SuhaylZhao** [comment]
Hi @songruo ,

The circuit diagram can be found here: 
https://github.com/Freenove/Freenove_ESP32_WROVER_Board/blob/main/Datasheet/ESP32_Schematic.pdf

**Comment by Alvin1Zhang** [comment]
Thanks for reporting and sorry for late reply, since this should be a hardware problem with the development board, you probably need to ask freenove for help, or find help on the forum https://esp32.com, will close the issue. Thanks.
